### ISSUE TITLE
Clicking on assignee name when editing issue takes you to the assigned issues filter

### ISSUE THREAD

**Issue by adamreisnz** [ZenHubIO/support#1344]
I don't know if this is intentional behaviour or not, but I think it didn't use to be like this, and it's quite frustrating.

Whenever I'm editing an issue and I want to change the assignee, I am used to clicking on either the assignee name or avatar. Clicking on the avatar is fine, but when I click on the name, the page now takes me to the issues filter for that assignee, closing the modal of the issue and potentially losing all the changes I've made to it.

I then have to go back to the board view and apply any filters I had on again, and find my issue, etc.

In short, it really breaks the whole workflow and has the potential for losing changes. 

Can you capture the click on the member name and show the assignee dropdown instead, just as happens when clicking on the avatar?

For clarity, I'm talking about this name link (in blue):

![image](https://user-images.githubusercontent.com/490562/35756508-8d9a664a-08d0-11e8-9bce-60000f50e4c9.png)

**Comment by kendramoroz** [comment]
Hey @adamreisnz when the assignee is already assigned, clicking on the name to filter is GitHub's behaviour at the moment to head to the filtered Issue list:

I did a few test scenarios with and without ZenHub active and it should always direct you to the Issue filter list

![screen shot 2018-02-02 at 2 03 45 pm](https://user-images.githubusercontent.com/11036230/35756940-f59c3cfa-0821-11e8-9ead-357bcd8d3abc.png)

Just to clarify what you're suggesting, are you attempting to click on the already assigned name to _edit_ the assignee? Currently this is reserved for the assignee copy itself, to activate the dropdown. 

Or, are you looking to have the `back` button to head back to the Boards remember your filter set?

When editing an Issue, it should remember the copy changes that are unpublished as long as you view it where you were editing it (opening an unsaved comment on the modal should still have that comment there) so you shouldn't be losing work!

**Comment by adamreisnz** [comment]
> I did a few test scenarios with and without ZenHub active and it should always direct you to the Issue filter list

Yes, but I'm 99% sure that this wasn't the case in one of your previous versions, as I've noticed it happening to me about 3 or 4 times in the last day, and that's usually a sign that something has changed, because I'm likely still doing the same action on my end.

We use zenhub a lot, so when changes like this happen, it's hard to miss them ;)

> Just to clarify what you're suggesting, are you attempting to click on the already assigned name to edit the assignee?

Yes that is correct. Again, I think this was already the case in a previous version and for some reason has been reverted. Or maybe the clickable area has changed, making it easier to misclick the avatar and click on the link instead -- it must be one of those two scenario's.

> When editing an Issue, it should remember the copy changes that are unpublished as long as you view it where you were editing it (opening an unsaved comment on the modal should still have that comment there) so you shouldn't be losing work!

Well this clearly doesn't work in this scenario. When I open an issue, then edit the description, then click on the assignee name and head to to filtered issues list, then back to the board and open the issue again, my changes are gone.

I think this isn't a good UX.

**Comment by kendramoroz** [comment]
Hey @adamreisnz thanks for the clarification!

There's been no changes on our end to the Assignee click-ability lately! I'd recommend reaching out to GitHub to confirm changes made on their end as this is the same behaviour when ZenHub is not active (Assignee names bumping you to the issue list, filtered). 

As for the edits in the description not saving, I'll look into this further to ensure we're remembering things the right away!

**Comment by adamreisnz** [comment]
Well, Zenhub already overrides many things that Github does, so can you perhaps look into making the name toggle the assignees dropdown as well? It's really frustrating when you're editing issues and suddenly you're being taken away to another screen.

The avatar and name are positioned very close to each other, so it's easy to misclick as well, even if you're aiming for the avatar.
